packages/pigeon/test/objc_generator_test.dart

```diff
@@ -347,8 +347,7 @@ void main() {
     generateObjcSource(const ObjcOptions(header: 'foo.h'), root, sink);
     final String code = sink.toString();
     expect(code, contains('@implementation Foobar'));
-    expect(code,
-        contains('pigeonResult.aBool = GetNullableObject(dict, @"aBool");'));
```

Comment:
Note: related to the other comment with respect to GetNullableObject.

Reply:
In this case, the list would already contain Null, no need to check if it's null since the result is the same 

(the function would say is this null? ok lets make it null instead)